libraries/AP_RangeFinder/AP_RangeFinder_MAVLink.cpp

```diff
@@ -55,8 +28,10 @@ void AP_RangeFinder_MAVLink::handle_msg(const mavlink_message_t &msg)
     if (packet.orientation == MAV_SENSOR_ROTATION_PITCH_270) {
         state.last_reading_ms = AP_HAL::millis();
         distance_cm = packet.current_distance;
+        _max_distance_cm = packet.max_distance;
+        _min_distance_cm = packet.min_distance;
```

Comment:
should use larger of the minimums, and smaller of the maximums
also, do we need a special case for magic values like 0 or UINT16_MAX?

Reply:
Done.

No magic values in this message.